### ISSUE TITLE
Sharing a full screen link on Firefox 60.6.1esr squashes graph

### ISSUE THREAD

**Issue by idbennm** [grafana/grafana#16296]
**What happened**:
Upgraded Firefox to version 20.6.1esr and since then if you view an individual graph in full screen then create a shareable link when you open the link the graph is sqaushed and unreadable - if you change the date range it displays fine - refreshing window does not
![image](https://user-images.githubusercontent.com/18311036/55232391-55e34e00-521d-11e9-98c4-9a6f21f4be54.png)

**What you expected to happen**:
Full screen to be displayed without squashing graph

**How to reproduce it (as minimally and precisely as possible)**:
- make sure you are running firefox 60.6.1esr (reproducible on windows 10 and Centos 7)
- Hover over an individual graph click on down arrow to select "View"/make the graph full screen
- click down arrow on title of graph and select "share"
- click copy 
- re-paste copied link into a new tab - graph does not render properly and is squashed 
- Change date range on graph to display properly 

**Anything else we need to know?**:

**Environment**:
- Grafana version:
6.0.2
- Data source type & version:
prometheus 2.8.0
- OS Grafana is installed on:
Centos 6
- User OS & Browser:
Seen on Windows 10 and Centos 7
- Grafana plugins:
N/A
- Others:
Same on graphs produced from metrics via multiple exporters

**Comment by tomponline** [comment]
This can be re-created by simply refreshing the page when viewing a single graph.

**Comment by marefr** [comment]
Have tried to reproduce this using Windows 10 and Firefox 60.6.1esr but to no avail. Could you please include more information. Screeenshots/animated gifs and dashboard json of panel you're viewing.

**Comment by idbennm** [comment]
Hi Thanks for replying so quickly please see below
![graphdisplay](https://user-images.githubusercontent.com/18311036/55237214-8af59d80-5229-11e9-9763-0b156583cecf.gif)

**Comment by idbennm** [comment]
This is working fine on previous versions of firefox

**Comment by marefr** [comment]
Thanks. Do you get any errors logged? Web developer > tools -> console

**Comment by tomponline** [comment]
@marefr yes indeed, this is on FF 60.6.0esr on centos 7

```
Error: Invalid dimensions for plot, width = 2508, height = 0
Stack trace:
n.prototype.resize@http://rps-mon.ictnw.net/grafana/public/build/app.8ec106c095469ac98ef1.js:5:48048
n@http://rps-mon.ictnw.net/grafana/public/build/app.8ec106c095469ac98ef1.js:5:17468
a/<@http://rps-mon.ictnw.net/grafana/public/build/app.8ec106c095469ac98ef1.js:5:43744
a@http://rps-mon.ictnw.net/grafana/public/build/app.8ec106c095469ac98ef1.js:5:43638
e.plot@http://rps-mon.ictnw.net/grafana/public/build/app.8ec106c095469ac98ef1.js:5:50692
m@http://rps-mon.ictnw.net/grafana/public/build/app.8ec106c095469ac98ef1.js:1:769319
g@http://rps-mon.ictnw.net/grafana/public/build/app.8ec106c095469ac98ef1.js:1:770226
ls</t.prototype.link/<@http://rps-mon.ictnw.net/grafana/public/build/app.8ec106c095469ac98ef1.js:1:770885
s.prototype.emit@http://rps-mon.ictnw.net/grafana/public/build/vendor.8ec106c095469ac98ef1.js:90:761031
r</e.prototype.emit@http://rps-mon.ictnw.net/grafana/public/build/app.8ec106c095469ac98ef1.js:1:819985
B</e.prototype.render@http://rps-mon.ictnw.net/grafana/public/build/app.8ec106c095469ac98ef1.js:1:237276
ls</t.prototype.onDataReceived@http://rps-mon.ictnw.net/grafana/public/build/app.8ec106c095469ac98ef1.js:1:762175
ls</t.prototype.onDataError@http://rps-mon.ictnw.net/grafana/public/build/app.8ec106c095469ac98ef1.js:1:761819
s.prototype.emit@http://rps-mon.ictnw.net/grafana/public/build/vendor.8ec106c095469ac98ef1.js:90:760601
r</e.prototype.emit@http://rps-mon.ictnw.net/grafana/public/build/app.8ec106c095469ac98ef1.js:1:819985
G</t.prototype.onMetricsPanelRefresh/<@http://rps-mon.ictnw.net/grafana/public/build/app.8ec106c095469ac98ef1.js:1:239703
c/</<@http://rps-mon.ictnw.net/grafana/public/build/vendor.8ec106c095469ac98ef1.js:134:92198
c/<@http://rps-mon.ictnw.net/grafana/public/build/vendor.8ec106c095469ac98ef1.js:134:92043
$digest@http://rps-mon.ictnw.net/grafana/public/build/vendor.8ec106c095469ac98ef1.js:134:97575
$apply@http://rps-mon.ictnw.net/grafana/public/build/vendor.8ec106c095469ac98ef1.js:134:99590
$applyAsync/r<@http://rps-mon.ictnw.net/grafana/public/build/vendor.8ec106c095469ac98ef1.js:134:99735
h@http://rps-mon.ictnw.net/grafana/public/build/vendor.8ec106c095469ac98ef1.js:134:33036
ln/this.$get</</i.defer/n<@http://rps-mon.ictnw.net/grafana/public/build/vendor.8ec106c095469ac98ef1.js:134:34396
 Possibly unhandled rejection: {} angular.js:14700
```

**Comment by mmwagner02** [comment]
I've been seeing essentially this same error. 

We've been using Firefox 60.x for a while and never had any problems with Grafana 5.x;  Updating to Grafana 6.x has caused the "invalid dimensions for plot" error when viewing individual panels in full screen.

We have Grafana installed on Centos 6 and RHEL7, and the problem is present on both.  

The problem does not occur in Chrome.

A non-identical but potentially related problem occurs in Internet Explorer.  In IE, when you try to view a panel fullscreen, it renders too big and won't fit on the screen,

**Comment by marefr** [comment]
Can you upgrade to latest Firefox version?

**Comment by tomponline** [comment]
Firefox 60 is the latest ESR release used in enterprises.

**Comment by mmwagner02** [comment]
I cannot reproduce the problem when using Firefox 66.0.2.
However, I'm in a corporate environment where we can only use ESR releases.  That limits us to FF 60.

**Comment by torkelo** [comment]
This issue should be fixed in v6.1

**Comment by tomponline** [comment]
Nice one thanks
